fractal_server/config.py

```diff
@@ -137,14 +141,16 @@ def DATABASE_URL(self):
     @property
     def DATABASE_SYNC_URL(self):
         if self.DB_ENGINE == "sqlite":
+            if not self.SQLITE_PATH:
+                raise ValueError("SQLITE_PATH path cannot be None")
             return self.DATABASE_URL.replace("aiosqlite", "pysqlite")
         elif self.DB_ENGINE == "postgres":
             return self.DATABASE_URL.replace("asyncpg", "psycopg2")
 
     ###########################################################################
     # FRACTAL SPECIFIC
     ###########################################################################
-    # FRACTAL_ROOT: Path = Path("FRACTAL_ROOT")
+    FRACTAL_ROOT: Path = _DEVNULL
```

Comment:
Any special reason for the difference between `FRACTAL_ROOT` (default: `_DEVNULL`) and `SQLITE_PATH` (default: `None`)?

Reply:
No, you're right. Will simplify this by making FRACTAL_ROOT an optional path.